test/parallel_tests/test_parallel_numpy.py

```diff
@@ -0,0 +1,126 @@
+import numpy
+import pytest
+from numpy.lib.format import (
+    _write_array_header,
+    header_data_from_array_1_0,
+)
+
+import lyncs_io as io
+from lyncs_io import numpy as np
+from helpers import *
+
+
+@pytest.mark.mpi_skip()
+def test_numpy_serial_load():
```

Comment:
Why do we have serial load/save here?
Isn't the same as those in test/test_numpy.py?

Reply:
It is just to ensure that the load/save routines could fall back to the serial version but you're right. That might be unecessary.

Reply:
Yeah this is checked by the standard serial tests. I would remove these